api/.env-example

```diff
@@ -4,11 +4,11 @@ PORT=3001
 # Assuming that you started your MySQL container with the below command:
 # docker run --name mysql -e MYSQL_ROOT_PASSWORD=mysql-root-password -e MYSQL_DATABASE=my-database -d -p 3306:3306 mysql:latest
 DB_CLIENT=mysql2
-DB_HOST=localhost
+DB_HOST=127.0.0.1
 DB_PORT=3306
 DB_USER=root
-DB_PASSWORD=mysql-root-password
-DB_DATABASE_NAME=meal-sharing
+DB_PASSWORD=pass
+DB_DATABASE_NAME=meal_sharing
```

Comment:
It is enough to edit this in the `.env` file, here you can keep the original values of the template.